man/crystal.1

```diff
@@ -0,0 +1,230 @@
+.\"Crystal Programming Language
+.Dd March 7, 2016
+.Dt CRYSTAL(1) "" "Crystal Compiler Command Line Reference Guide"
+.\".Dt CRYSTAL 1
+.Os UNIX
+.Sh NAME
+.Nm crystal - Compiler for the Crystal language.
+.Sh SYNOPSYS
+.Nm
+command
+.Op switches
+programfile
+--
+.Op arguments
+.Sh DESCRIPTION
+Crystal is a statically type-checked programming language. It was created with the beauty of ruby and the performance of pure C in mind.
+.Sh USAGE
+You can compile and run a program by invoking the compiler with a single filename:
+.Bd -offset indent-two
+.Nm
+some_program.cr
+.Ed
+
+Crystal files usually end with the .cr extension.
+
+Alternatively you can use the run command:
+.Bd -offset indent-two
+.Nm
+run
+some_program.cr
+.Ed
+
+To create an executable use the build command:
+.Bd -offset indent-two
+.Nm
+build
+some_program.cr
+.Ed
+
+This will create an executable named "some_program".
+
+Note that by default the generated executables are not fully optimized.
+To turn optimizations on, use the --release flag:
+.Bd -offset indent-two
+.Nm
+--release
+build
+some_program.cr
+.Ed
+
+Make sure to always use --release for production-ready executables and when performing benchmarks.
+
+The reason for this is that performance without full optimizations is still pretty good and provides fast compile times, so you can use the crystal command almost as if it were an interpreter.
+
+
+.Bl -tag -width "12345678" -compact
+.Pp
+.Sh OPTIONS
+The crystal command accepts the following options 
+
+.Bl -tag -width "12345678" -compact
+.Pp
+.It 
+.Cm init
+TYPE
+NAME
+.Op DIR
+.Pp
+Generate a new project.
+.Pp
+TYPE is one of:
+.Bl -tag -width "12345678" -compact
+.Pp
+.It Sy lib
+Creates a library skeleton
+.It Sy app
+Creates an application skeleton
+.El
+.Bd -literal -offset
+NAME - name of project to be generated, eg: example
+.Pp
+DIR  - directory where project will be generated, default: NAME
+.Ed
+
+.Pp
+.It
+.Cm build
+.Op options
+programfile
+--
+.Op arguments
+.Pp
+Compile program.
+.Pp
+Options:
+.Bl -tag -width "12345678" -compact
+.Pp
+.It Fl -cross-compile
+Generate object files for cross compilation and prints the command to build the executable.
+.It Fl -d, Fl -debug
+Generate the output with symbolic debug info.
```

Comment:
"with debug symbols."

"These are read when debugging the built program with tools like lldb, gdb, valgrind etc. and provide mappings to the original source code for those tools."